flexmeasures/ui/templates/crud/assets.html

```diff
@@ -61,7 +61,13 @@
         
         let url = `${basePath}/api/v3_0/assets?page=${
           Math.floor(data["start"] / data["length"]) + 1
-        }&per_page=${data["length"]}&all_accessible=true`;
+        }&per_page=${data["length"]}`;
+
+        if ("{{ account }}" != "None") {
```

Comment:
This should probably better be a Jinja2 if/else, which can do a real check for  `account is None`, not a JavaScript one checking for the literal string "None".